src/main/java/org/fogbowcloud/saps/engine/core/pojo/ImageTaskFile.java

```diff
@@ -0,0 +1,48 @@
+package org.fogbowcloud.saps.engine.core.pojo;
+
+import org.fogbowcloud.saps.engine.core.model.ImageTask;
+
+/**
+ * Holds the information of a file generated by the processing of a {@link ImageTask}.
+ */
+public class ImageTaskFile {
+
+    private String path;
+    private String name;
+    private String URL;
+
+    public ImageTaskFile(String path, String name) {
```

Comment:
do we need (sorry, I did not checkout the code) both these constructor (can we keep only one?)

Reply:
The other constructor isn't used externally, I added just for precaution, so yes, it can be removed